recipe/build.sh

```diff
@@ -25,8 +89,6 @@ if [ $(uname) == Linux ]; then
                 -headerdir $PREFIX/include/qt \
                 -archdatadir $PREFIX \
                 -datadir $PREFIX \
-                -L $PREFIX/lib \
-                -I $PREFIX/include \
```

Comment:
And this removal?

Reply:
See above. (Marking this as "resolved" now since this and the above are identical points of discussion.)